docs/guides/build_and_publish.md

```diff
@@ -1,19 +1,21 @@
-# Publishing a package
+# Building and publishing a package
 
-uv supports building Python packages into source and binary distributions via `uv build` and
-uploading them to a registry with `uv publish`.
+uv supports building Python packages into source and binary distributions via the `uv` build
+backend, the `uv build` build frontend and uploading them to a registry with `uv publish`.
```

Comment:
I think the original text sounded better. The new text makes it sound like `uv` supports building _only_ via the `uv` build backend (even though "only" is not written).